shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -888,11 +949,25 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:@"TextInput.clearClient"]) {
     [self clearTextInputClient];
     result(nil);
+  } else if ([method isEqualToString:@"TextInput.setEditableSizeAndTransform"]) {
+    [self setEditableSizeAndTransform:args];
+  } else if ([method isEqualToString:@"TextInput.setMarkedTextRect"]) {
+    [self updateMarkedRect:args];
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+- (void)setEditableSizeAndTransform:(NSDictionary*)dictionary {
+  [_activeView setEditableTransform:dictionary[@"transform"]];
+}
+
+- (void)updateMarkedRect:(NSDictionary*)dictionary {
+  CGRect rect = CGRectMake([dictionary[@"x"] doubleValue], [dictionary[@"y"] doubleValue],
```

Comment:
fyi you'll end up with CGRectZero if the dictionary is empty or nil.  I'm not sure if that's expected.

Reply:
Added an assert